packages/contracts-bedrock/test/legacy/L1ChugSplashProxy.t.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.15;
+
+// Testing utilities
+import { Test } from "forge-std/Test.sol";
+
+// Target contract
+import { L1ChugSplashProxy } from "src/legacy/L1ChugSplashProxy.sol";
+
+contract L1ChugSplashProxyWrapper is L1ChugSplashProxy {
```

Comment:
I think we can get rid of this, all that we need here is the deploy code prefix so we can just copy/paste it.